{
  "class": "vulnerability",
  "id": "oval:com.ubuntu.kinetic:def:2021207180000000",
  "version": "1",
  "metadata": {
    "Title": "CVE-2021-20718 on Ubuntu 22.10 (kinetic) - medium.",
    "Description": "mod_auth_openidc 2.4.0 to 2.4.7 allows a remote attacker to cause a denial-of-service (DoS) condition via unspecified vectors.\n\nUpdate Instructions:\n\nRun `sudo pro fix CVE-2021-20718` to fix the vulnerability. The problem can be corrected\nby updating your system to the following package versions:\n\nlibapache2-mod-auth-openidc - 2.4.9-1\nNo subscription required",
    "affected": {
      "family": "unix",
      "Platform": "Ubuntu 22.10"
    },
    "reference": {
      "source": "CVE",
      "ref_id": "CVE-2021-20718",
      "ref_url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-20718"
    },
    "advisory": {
      "Severity": "Medium",
      "Rights": "Copyright (C) 2021 Canonical Ltd.",
      "PublicDate": "2021-05-20 02:15:00 UTC",
      "Bug": null,
      "cve": {
        "text": "CVE-2021-20718",
        "href": "https://ubuntu.com/security/CVE-2021-20718",
        "severity": "medium",
        "public": "20210520",
        "cvss_score": "7.5",
        "cvss_vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
      },
      "PublicDateAtUsn": "",
      "AssignedTo": "",
      "DiscoveredBy": "",
      "Crd": "",
      "Ref": null
    }
  },
  "notes": {
    "Note": ""
  },
  "criteria": {
    "criterions": [
      {
        "test_ref": "oval:com.ubuntu.kinetic:tst:2021207180000000",
        "comment": "libapache2-mod-auth-openidc package in kinetic, is related to the CVE in some way and has been fixed (note: '2.4.9-1')."
      }
    ]
  }
}
